aws-android-sdk-core/src/main/java/com/amazonaws/logging/LogFactory.java

```diff
@@ -53,20 +54,18 @@ public static synchronized Log getLog(String logTag) {
         logTag = getTruncatedLogTag(logTag);
 
         Log log = logMap.get(logTag);
-        if (log == null) {
-            if (checkApacheCommonsLoggingExists()) {
-                try {
-                    log = new ApacheCommonsLogging(logTag);
-                    logMap.put(logTag, log);
-                } catch (Exception e) {
-                    android.util.Log.w(TAG, "Could not create log from " + APACHE_COMMONS_LOGGING_LOGFACTORY, e);
-                }
-            }
-            if (log == null) {
-                log = new AndroidLog(logTag);
-                logMap.put(logTag, log);
-            }
+        if (log != null) {
+            return log;
+        }
+
+        if (checkApacheCommonsLoggingExists()) {
```

Comment:
I would just get of any semblance of support for Apache.

Reply:
I will do this in a separate PR in case there is even the smallest chance of needing to revert the commit :)
Will be followed up by #2277